PIMbench/util.h

```diff
@@ -21,14 +21,16 @@
 
 using namespace std;
 
+#define MAX_NUMBER 1024
+
 void getVector(uint64_t vectorLength, std::vector<int> &srcVector)
 {
   srand((unsigned)time(NULL));
   srcVector.resize(vectorLength);
 #pragma omp parallel for
   for (uint64_t i = 0; i < vectorLength; ++i)
   {
-    srcVector[i] = (rand() % (i + 1) + 1);
+    srcVector[i] = (rand() % MAX_NUMBER);
```

Comment:
Do we also need to update getMatrix() function  as above..

Reply:
Updated thanks!